readme/template-guide-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template Guide
+
+Read this document before you start working on the README template.
+
+
+*Note: As your project evolves and grows, so should your README file. READMEs should be revisited regularly to ensure relevant information is captured for new users.*
+
+## Why do I need a README?
+READMes are the first document users find and read. All projects include a README, and users will expect to find one in your project's root folder. A README tells users what they need to know about your project, if the project is relevant to the user, and how users can engage with the project. Your README is your project's first impression.
+
+In some cases, larger projects may include sub-READMEs, or separate, smaller READMEs that support subfolders in the project.
+
+## Content of the README template guide
+The following sections provide guidance and context on how to fill out the README template.
+
+### About the "Project logo and badge" section
+This section is optional. 
+
+A project logo can be helpful for users to visually identify your project. 
+
+Embed relevant badges to your README to add credibility to your project. Badges highlight important information and can help users determine project fit quickly. [Learn more about badges.](https://github.com/badges/shields)
+
+### About the "Project name" section
```

Comment:
You use the capitalisation "Project Name" in the template, should this title reflect that?

Reply:
Updated!